libs/utils/include/utils/Log.h

```diff
@@ -97,7 +97,7 @@ class ostream {
     const char* getFormat(type t) const noexcept;
 
     class Buffer {
-        char mStorage[1024];
+        char mStorage[16384];
```

Comment:
Should our log be this large in all build configurations?  With emscripten, memory is very precious.